sys/net/gnrc/application_layer/tftp/gnrc_tftp.c

```diff
@@ -1047,6 +1047,9 @@ int _tftp_decode_start(tftp_context_t *ctxt, uint8_t *buf, gnrc_pktsnip_t *outbu
 
     /* decode the TFTP transfer mode */
     for (uint32_t idx = 0; idx < ARRAY_LEN(_tftp_modes); ++idx) {
+        if (_tftp_modes[idx].len > (len - sizeof(*hdr) - fnlen))
```

Comment:
coding style requires braces even for one-line `if`s

Reply:
Oops, fixed.